db/migrate/20210616103631_create_npq_profiles.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateNpqProfiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :npq_profiles do |t|
+      t.references :user
+      t.references :npq_lead_provider
+      t.references :npq_course
+
+      t.date :date_of_birth
+      t.text :teacher_reference_number
+      t.boolean :teacher_reference_number_verified, default: false
```

Comment:
this will be the marker used to determine if application goes into the "holding pen"